packages/core-flows/src/definition/cart/steps/find-one-or-any-region.ts

```diff
@@ -0,0 +1,27 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IRegionModuleService } from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+export const findOneOrAnyRegionStepId = "find-one-or-any-region"
+export const findOneOrAnyRegionStep = createStep(
+  findOneOrAnyRegionStepId,
+  async (data: { regionId?: string }, { container }) => {
+    const service = container.resolve<IRegionModuleService>(
+      ModuleRegistrationName.REGION
+    )
+
+    if (!data.regionId) {
+      const regions = await service.list({})
```

Comment:
Making this deterministic by eg. sorting (maybe there already is some default sorting?) would be a bit better IMO.

Reply:
there is no default sorting indeed a default would be nice to have one indeed.

Reply:
Adding a ticket for this @adrien2p